From 6279c6265485a15e1655479252d56b90cd53cfbc Mon Sep 17 00:00:00 2001
From: Pascal Paillet <p.paillet@foss.st.com>
Date: Mon, 2 Sep 2024 09:18:34 +0200
Subject: [PATCH 1116/1695] dts: stm32: remove export vddgpu on stm32mp25

Remove the export of vddgpu as this is now managed
by a power domain.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git ca388ebd224eeefa15a622f5038240a87076ba9f dts: stm32: remove export vddgpu on stm32mp25]
Change-Id: I914976169bc5c846d13df5eb44f455447dc4dbcb
Signed-off-by: Pascal Paillet <p.paillet@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/401353
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Reviewed-by: Thomas BOURGOIN <thomas.bourgoin@foss.st.com>
Domain-Review: Patrick DELAUNAY <patrick.delaunay@foss.st.com>
Tested-by: Pascal PAILLET <p.paillet@st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Pascal PAILLET <p.paillet@st.com>
---
 core/arch/arm/dts/stm32mp251.dtsi | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/core/arch/arm/dts/stm32mp251.dtsi b/core/arch/arm/dts/stm32mp251.dtsi
index 3b78050a6..9176ea25f 100644
--- a/core/arch/arm/dts/stm32mp251.dtsi
+++ b/core/arch/arm/dts/stm32mp251.dtsi
@@ -162,10 +162,6 @@
 			reg = <VOLTD_SCMI_ADC>;
 			voltd-supply = <&vdda18adc>;
 		};
-		voltd-vddgpu {
-			reg = <VOLTD_SCMI_GPU>;
-			voltd-supply = <&vddgpu>;
-		};
 	};
 
 	scmi_cm33_agent: scmi-cm-agent@0 {
-- 
2.47.2

